uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/Entitlement.php

```diff
@@ -1,249 +0,0 @@
-<?php
```

Comment:
I am not sure what's going on here .. why is this entity get removed ? is this because the new data structure ?